docs/_templates/partials/header.html

```diff
@@ -85,22 +85,25 @@
                     {{ alt.name }}
                   </a>
                 </li>
-                {% endfor %}
+              {% endfor %}
             </ul>
           </div>
         </div>
       </div>
     {% endif %}
 
-    <!-- Button to open search modal -->
-    {% if "search" in config["plugins"] %}
-      <label class="md-header__button md-icon" for="__search">
-        {% include ".icons/material/magnify.svg" %}
-      </label>
-
-      <!-- Search interface -->
-      {% include "partials/search.html" %}
-    {% endif %}
+    <!-- Algolia search -->
```

Comment:
Would it be possible to make the fonts in the search results much larger? It's a bit hard for me to read. (I know we said not to worry about design, but I've changed my mind on this one aspect.)

Reply:
I think we can make it even the same as it was before, but we will need to update our CSS files
If you want - I can do that, but it will take some time
Or also yes, I can increate just the text size also